### DISCUSSION TITLE
能不能有什么办法防一放这个

### DISCUSSION THREAD

**Discussion by ning0818** [Qexo/Qexo#311] [Ideas]
虽然可以一键清楚，但是最好还是防一防

![image](https://user-images.githubusercontent.com/117955401/215402672-788a3a8c-bf57-477d-bfa5-17756ee2cb18.png)

**Comment by sysfox** [comment]
好家伙，这是被刷接口了？

**Comment by sysfox** [comment]
Ps：标题打错了（doge
@ning0818

**Reply by ning0818** [reply]
哦对，但好像不能改标题

**Reply by sysfox** [reply]
标题右侧有一个edit，还有你正文打成了“放一放”

**Reply by sysfox** [reply]
被刷接口先用谷歌验证防一下吧

**Reply by ning0818** [reply]
对，edit只能改内容，改不了标题

**Comment by sysfox** [comment]
最讨厌这种刷接口的脚本小子，所以这个功能我都不用的

**Comment by am-abudu** [comment]
我也被刷过，所以添加了验证码功能。友联申请可以开启验证码的

**Reply by ning0818** [reply]
recaptcha?我再注册gmail的时候告诉我手机号不可用于验证

**Comment by Fgaoxing** [comment]
后面我搞，我在忙，你可以先把他关掉，<del>不过你这种抄袭，被刷很正常，我还被DDOS过，不知道得罪了谁</del>

**Comment by Fgaoxing** [comment]
qexo-friends已发布1.0.5，等缓存，新版理论支持recaptcha
